### DISCUSSION TITLE
Step 3 is not getting updated.

### DISCUSSION THREAD

**Discussion by methmal99** [skills/.github#265] [Q&A]
Already finished the step 2. But when refreshed, it didn't update to the step 3. Still it was at the step 2.

**Comment by sinsukehlab** [comment]
You mean the Introduction to GitHub course?
You have mistakenly committed to the `main` branch instead of the `my-first-branch` branch.

**Reply by sinsukehlab** [reply]
It's strange that Step 4 workflow took 24 minutes though.
